src/main/java/net/dv8tion/jda/api/EmbedBuilder.java

```diff
@@ -24,7 +25,7 @@
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
-import java.awt.Color;
+import java.awt.*;
```

Comment:
I don't think it's good to have java.awt.* here.

Reply:
Fixed.